src/configure/templateInputHelper/InputControl.ts

```diff
@@ -72,10 +72,10 @@ export class InputControl {
                 this.value = await this.dataSource.evaluateDataSources(dependentInputs, azureSession);
             }
             else if (this.controlType === ControlType.QuickPick) {
-                let selectedValue = await this.dataSource.evaluateDataSources(dependentInputs, azureSession)
-                    .then((listItems: Array<{ label: string, data: any, group?: string }>) => {
-                        return this.showQuickPick(this.getInputControlId(), listItems, { placeHolder: this.inputDescriptor.name });
-                    });
+                let selectedValue = await this.showQuickPick(this.getInputControlId(),
```

Comment:
If I am not wrong, this was a window notification. Are you changing to quickpick?